rosetta/configuration/configuration.go

```diff
@@ -59,15 +57,15 @@ func New(chain flow.ChainID) *Configuration {
 	}
 
 	errors := []meta.ErrorDefinition{
-		ErrorInternal,
-		ErrorInvalidFormat,
-		ErrorInvalidNetwork,
-		ErrorInvalidAccount,
-		ErrorInvalidCurrency,
-		ErrorInvalidBlock,
-		ErrorInvalidTransaction,
-		ErrorUnknownBlock,
-		ErrorUnknownCurrency,
+		errors.ErrorInternal,
+		errors.ErrorInvalidFormat,
+		errors.ErrorInvalidNetwork,
+		errors.ErrorInvalidAccount,
+		errors.ErrorInvalidCurrency,
+		errors.ErrorInvalidBlock,
+		errors.ErrorInvalidTransaction,
+		errors.ErrorUnknownBlock,
+		errors.ErrorUnknownCurrency,
```

Comment:
I dislike this change, because the `configuration` package neatly collected everything that is part of the Rosetta implementation's hard-coded configuration in a single place. Now, it is distributed over multiple places, and we mix up the _configuration_ with error types/definitions and the actual _implementation_ with its _error handling_.